### DISCUSSION TITLE
LapTime data from ergast

### DISCUSSION THREAD

**Discussion by harish5p** [theOehrly/Fast-F1#153] [Q&A] [answered]
Hello, 

I'm currently using `session = fastf1.get_session(2022, 3, "R")
laps = session.load_laps()` to get the LapTime data for each driver, but since some of these have  `IsAccurate` as False I have to exclude them. I'm wondering if I can get the data from ergast which has accurate data including all the laps. My memory is fuzzy but I remember reading some of the data is taken from ergast. I've also tried using the `force_ergast=True` but the laptimes are still not accurate. Thanks for this amazing package.

**Comment by theOehrly** [comment, marked as answer]
'IsAccurate' does not refer to lap times. Those are assumed to be 100% correct. The 'Time' and 'LapStartTime' might be slightly offset, but not necessarily.
